backend/game/cards/keep_cards/health_manipulation_cards/test_even_bigger.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from game.cards.keep_cards.health_manipulation_cards.even_bigger import EvenBigger
+from game.player.player import Player
+from game.values import constants
+
+
+@pytest.fixture(autouse=True)
```

Comment:
you do not need to add this fixture/def here

Reply:
I did this so I didn't have to keep doing the add player, add the card to the player, then trigger the immediate effect. Should I just have done that in each test?